mx.pwmc.Svgvi.json

```diff
@@ -0,0 +1,123 @@
+{
+    "app-id": "mx.pwmc.Svgvi",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.30",
+    "sdk": "org.gnome.Sdk",
+    "writable-sdk": true,
+    "command": "mx.pwmc.Svgvi",
+    "finish-args": [
+        "--share=network",
```

Comment:
What does it use networking for?

Reply:
Any file on the network can be opened for edition, thanks to GIO.